.buildkite/scripts/steps/checks/verify_fips_enabled.sh

```diff
@@ -2,7 +2,10 @@
 
 set -euo pipefail
 
-source .buildkite/scripts/common/util.sh
+if [[ -z "${BASH_SOURCE[1]+x}" || "${BASH_SOURCE[1]}" != *"checks.sh"* ]]; then
```

Comment:
Any context on this?

Reply:
If this script is called from `checks.sh` the bootstrap has already run, but in this pipeline we're calling the script directly and it relies on bootstrap. I'll add a comment.